### PR TITLE
Use setup.cfg for aiidalab metadata declaration.

### PR DESCRIPTION
**Pull request by csadorf**

### CODE REVIEW

**Comment by csadorf** [comment]
See https://aiidalab.readthedocs.io/en/latest/app_development/publish.html for an explanation regarding this change.

File: .aiidalab/setup.cfg [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[aiidalab]
+title = OPTIMADE Client
+authors = Casper Welzel Andersen
+description = AiiDAlab app to query for and import structures from OPTIMADE providers (COD, Materials Cloud, NoMaD, Materials Project, OQMD, and more ...).
+external_url = https://www.optimade.org
+documentation_url = https://github.com/CasperWA/voila-optimade-client#readme
+state = stable
+categories = utility
```
Comment (CasperWA) [inline .aiidalab/setup.cfg]:
```suggestion
categories = utilities
```
It seems [here](https://github.com/aiidalab/aiidalab-registry/blob/master/categories.yaml) that the value is the plural version and not singular. Maybe that should be unified to be plural/singular?

Comment (csadorf) [reply .aiidalab/setup.cfg]: Indeed probably better to unify.

**Review by CasperWA — APPROVED** [review]
Thanks a bunch @csadorf ! :)

**Pull request merged**